.github/workflows/test.yaml

```diff
@@ -400,6 +400,7 @@ jobs:
         - ubuntu-core-18-64
         - ubuntu-core-20-64
         - ubuntu-core-22-64
+        - ubuntu-core-22-arm-64
```

Comment:
I'd prefer arm64 to match the debian architecture. I think that we are in general more used to that.

Reply:
Already discussed. It will remain like this to match systems in other backends